src/libraries/adaptive-curve/ConstantsLib.sol

```diff
@@ -5,15 +5,21 @@ pragma solidity ^0.8.0;
 /// @author Morpho Labs
 /// @custom:contact security@morpho.org
 library ConstantsLib {
-    /// @notice Maximum rate at target per second (scaled by WAD) (1B% APR).
-    int256 internal constant MAX_RATE_AT_TARGET = int256(0.01e9 ether) / 365 days;
+    /// @notice Curve steepness (scaled by WAD).
+    int256 public constant CURVE_STEEPNESS = 4 ether;
 
-    /// @notice Mininimum rate at target per second (scaled by WAD) (0.1% APR).
-    int256 internal constant MIN_RATE_AT_TARGET = int256(0.001 ether) / 365 days;
+    /// @notice Adjustment speed per second (scaled by WAD).
+    int256 public constant ADJUSTMENT_SPEED = int256(50 ether) / 365 days;
```

Comment:
Note that part of the "old" `ADJUSTMENT_SPEED` natspec defined in `IAdaptiveCurveIrm` has not been moved here

> @dev The speed is per second, so the rate moves at a speed of ADJUSTMENT_SPEED * err each second (while being continuously compounded).